build/build.yml

```diff
@@ -51,6 +51,80 @@ steps:
       summaryFileLocation: $(System.DefaultWorkingDirectory)/coverage/cobertura-coverage.xml
       additionalCodeCoverageFiles: $(System.DefaultWorkingDirectory)/coverage-remapped/coverage.json
 
+  - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@1
```

Comment:
This shouldn't be done as a separate step - you can just add the signing operations to the one below.

(and all the entries in the file patterns list shouldn't be necessary. We're only signing the VSIX's here)

Reply:
Also - did you verify that authenticode signing doesn't affect the ability to install and run the extensions from VS Code?